WordPress/src/test/java/org/wordpress/android/ui/jetpack/BackupProgressCheckerTest.kt

```diff
@@ -20,18 +20,22 @@ import org.wordpress.android.fluxc.store.ActivityLogStore
 import org.wordpress.android.fluxc.store.ActivityLogStore.BackupDownloadStatusError
 import org.wordpress.android.fluxc.store.ActivityLogStore.BackupDownloadStatusErrorType
 import org.wordpress.android.fluxc.store.ActivityLogStore.OnBackupDownloadStatusFetched
+import org.wordpress.android.ui.jetpack.backup.BackupDownloadProgressChecker
 import java.util.Date
 
 @RunWith(MockitoJUnitRunner::class)
-class BackupDownloadProgressCheckerTest {
+class BackupProgressCheckerTest {
     @Mock lateinit var activityLogStore: ActivityLogStore
     @Mock lateinit var site: SiteModel
     private val restoreId = 1L
     private lateinit var backupDownloadProgressChecker: BackupDownloadProgressChecker
 
     @Before
     fun setUp() {
-        backupDownloadProgressChecker = BackupDownloadProgressChecker(activityLogStore, TEST_SCOPE)
+        backupDownloadProgressChecker = BackupDownloadProgressChecker(
```

Comment:
Did you intend to rename even the BackupDownloadProgressChecker?

Reply:
@malinajirka 
> Note 1: It seems there are still some places (classes and field names + FluxC) which use the "backupDownload" convention. Is there some logic behind it or do you plan to rename everything iteratively?

I am on the fence about renaming everything in FluxC because having `download` there provides additional context since the logic sits outside of WP. 

> Note 2: I don't want to suggest that we shouldn't rename it, I just want to share my point of view. When I read "BackupStatusService.start()" my first thought is that the service takes care of creating a backup (not that it prepares an already existing backup for download). Similar goes for BackupProgressCheckerTest. Wdyt? 

This has been rattling my brain since I named these. *Backup* is the language that everyone has been using when referring to  *preparing the downloadable backup file*, but like you said the backup is already there. I have been torn using ubiquitous language or coming up with a different name that feels clear (and is not verbose). I _thought_ BackupDownload was it, but it just keeps rubbing me the wrong way and no way does "download" feel like it should stand on it's own (absolutely no context there).. 🤦  Names are hard. I can just bite the bullet and use BackupDownload - It was my first choice and perhaps that instinct was correct.

Reply:
> I am on the fence about renaming everything in FluxC because having download there provides additional context since the logic sits outside of WP.

I see, make sense.

> This has been rattling my brain since I named these. Backup is the language that everyone has been using when referring to preparing the downloadable backup file, but like you said the backup is already there. I have been torn using ubiquitous language or coming up with a different name that feels clear (and is not verbose). I thought BackupDownload was it, but it just keeps rubbing me the wrong way and no way does "download" feel like it should stand on it's own (absolutely no context there).. 🤦 Names are hard. I can just bite the bullet and use BackupDownload - It was my first choice and perhaps that instinct was correct.

"Names are hard" - I completely agree :D. I think it might be easier to decide when we see all the classes/packages. So I'd personally postpone this decision for later and wouldn't worry about it now. For now, we can just consistently follow one of these approaches. Wdyt?

Reply:
> So I'd personally postpone this decision for later and wouldn't worry about it now. 

***YES***!! I changed the other classes back to BackupDownload in this PR and am moving forward with that. I'm leaving the package as Backup, but will use BackupDownload prefix for Activities, Frags, VM's and the rest. Going to merge and get back to my skeletons.